xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/inputstream/stream_codec.h

```diff
@@ -141,6 +141,28 @@ extern "C"
     /// [chroma subsampling](https://en.wikipedia.org/wiki/Chroma_subsampling): 4:2:0, 4:2:2, 4:4:4
     /// see [AV1 specification](https://aomedia.org/av1/specification/)
     AV1CodecProfileProfessional,
+
+    // Audio codec profiles
+
+    AACCodecProfileMAIN,
+    AACCodecProfileLOW,
+    AACCodecProfileSSR,
+    AACCodecProfileLTP,
+    AACCodecProfileHE,
+    AACCodecProfileHEV2,
+    AACCodecProfileLD,
+    AACCodecProfileELD,
+    MPEG2AACCodecProfileLOW,
+    MPEG2AACCodecProfileHE,
+    DTSCodecProfile,
+    DTSCodecProfileES,
+    DTSCodecProfile9624,
```

Comment:
Specially here, as firstly I become a big thinking in mind whats "9624" means :sweat_smile::thinking: and comes then the "96 kHz / 24 Bit" in mind.